Update OEE to 10.17 and Shifts feature. (#1238)
SoftwareAG/c8y-docs: content/oee/oee-administration-bundle/organization.md

### To assign an entity

> Click the **Add** button to add an entity (line, area, or machine) to another entity. It is possible to assign machines to multiple lines by adding them. This allows for modelling of setups where lines can split, join, or share equipment between them.

Reviewer: ```suggestion
Click the **Add** button to add an entity (line, area, or machine) to another entity. You can assign one or more machines to multiple lines. This allows for modelling of setups where lines can split, join, or share equipment between them.
```
